### PR TITLE
Rule for custom Chinese packer: ChornClickers

### PR DESCRIPTION
**Pull request by enovella**
Further info at https://github.com/rednaga/APKiD/issues/93

### CODE REVIEW

File: apkid/rules/apk/packers.yara [hunk 1]
```diff
@@ -534,3 +530,21 @@ rule apkpacker : packer
     condition:
         is_apk and all of them
 }
+
+rule chornclickers : packer
+{
+
+  meta:
+    description = "Custom Chinese 'ChornClickers'"
```
Comment (CalebFenton) [inline apkid/rules/apk/packers.yara]: "Custom Chinese" is accurate but super redundant :-P. They're all "custom chinese". How about just "ChornClickers" with _maybe_ comment explaining the name was invented by us

Comment (enovella) [reply apkid/rules/apk/packers.yara]: I dislike commenting too much the `description` tag. 

These are the two ways I've tried with:
```
[*] /media/edu/terita/apkid-todo/custom-chinese-packer-93/2ffc6fbe4e40417f35dbfb579c858d32
 |-> packer : ChornClickers

[*] /media/edu/terita/apkid-todo/custom-chinese-packer-93/2ffc6fbe4e40417f35dbfb579c858d32
 |-> packer : ChornClickers ('Ch'-china '-orn'-porn '-clickers')
```

Finally, I've decided to explain the origin in the source code like:
```
meta:
    description = "ChornClickers"
    explanation = "ChornClickers ('Ch'-china '-orn'-porn '-clickers')"
```

File: apkid/rules/apk/packers.yara [hunk 2]
```diff
@@ -534,3 +530,22 @@ rule apkpacker : packer
     condition:
         is_apk and all of them
 }
+
+rule chornclickers : packer
+{
+
+  meta:
+    description = "ChornClickers"
+    explanation = "ChornClickers ('Ch'-china '-orn'-porn '-clickers')"
```
Comment (CalebFenton) [inline apkid/rules/apk/packers.yara]: Sorry I wasn't more clear about this -- I just meant to add something like:
```// This has no name so we made one up from 'Ch'-china '-orn'-porn '-clickers'```

Comment (enovella) [reply apkid/rules/apk/packers.yara]: Done

**Pull request merged**